src/main/java/com/kimsquitsystem/kimsquitdemo/model/User.java

```diff
@@ -0,0 +1,106 @@
+package com.kimsquitsystem.kimsquitdemo.model;
+
+import java.util.HashMap;
+
+public class User {
```

Comment:
We need to discuss about having inherited users or just Teachers and Students

Reply:
students and teachers share a lot of the same attributes